enterprise/internal/campaigns/store/changesets.go

```diff
@@ -712,6 +719,49 @@ func scanChangesets(rows *sql.Rows, queryErr error) ([]*campaigns.Changeset, err
 	})
 }
 
+// jsonCampaignChangesetSet represents a "join table" set as a JSONB object
```

Comment:
I realized that join tables are, as far as I know, always two-sided: you can join A against B, and B against A, but this one here is more like an embedded has-one association, so maybe lower-expectations a bit? 😄 (Feel free to ignore though)

```suggestion
// jsonCampaignChangesetSet represents a "one-sided join table" set as a JSONB object
```

Reply:
I _think_ we use it in both ways, although that is probably not the best performance 😬